### DISCUSSION TITLE
How to convert code action view to normal message view?

### DISCUSSION THREAD

**Discussion by jeeeem** [folke/noice.nvim#445] [Q&A]
I want this code action notification to go back to default message since it is kinda hard to see what my choosing option.

I am not sure what route or redirection config should I add.

![image](https://user-images.githubusercontent.com/51053482/235057835-58293f19-af1c-413a-8c1b-5eeddca94903.png)

**Comment by yutkat** [comment]
```lua
require("noice").setup({
	messages = {
		enabled = false,
	},
})
```

or 

```lua
require("noice").setup({
	lsp = {
		message = {
			enabled = false,
		}
	},
})
```

**Comment by folke** [comment]
Better to use something like dressing.nvim for code actions

![image](https://user-images.githubusercontent.com/292349/235098440-979fd9f0-1181-473c-ba14-a868fac1a38b.png)

**Reply by NobbZ** [reply]
Do you have some config example how to use this for code actions, while `noice` for everything else?

To add on this:

Most of the time, noice does not display any code actions for me at all, until I hit `Esc` to cancel them, then I get a notification displaying my options.

It is irrelevant for this whether `lsp.message.enable` is set to `true` or `false`.

In the more rare occassions, I can see the notification from the start, though it will often decay before I can read over all the options. Hitting any number will delete the notification anyway and replace its content with my keys pressed.
